lighthouse-core/audits/deprecations.js

```diff
@@ -33,7 +33,7 @@ class Deprecations extends Audit {
     return {
       category: 'Deprecations',
       name: 'deprecations',
-      description: 'Site does not use deprecated APIs',
+      description: 'Avoids deprecated APIs',
```

Comment:
Yeah... given the constraints of the current UI, I don't see a better alternative to `avoid`! Although something doesn't sit well with me about it.

*Rides off grumbling into the sunset*